### PR TITLE
Newly Written Rule: Google Workspace Apps Marketplace Allowlist

### PR DESCRIPTION
**Pull request by andrea-youwakim**
### Background

<High level overview here>

### Changes

* Added newly written rule that monitors and alerts on events related to Google Workspace Apps Marketplace Allowlist changes.

### Testing

* 
Screen Shot 2022-12-15 at 12 47 13 PM

### CODE REVIEW

File: rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    setting_name = event.get("parameters", {}).get("SETTING_NAME", "NO_SETTING_NAME")
```
Comment (calkim-panther) [inline rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py:3]: I think we've been wrapping defaults in <>

File: rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    return setting_name == "ENABLE_G_SUITE_MARKETPLACE" and old_val != new_val
+
+
+def title(event):
+    # (Optional) Return a string which will be shown as the alert title.
+    # If no 'dedup' function is defined, the return value of this
+    # method will act as deduplication string.
+    value_dict = {
+        "DEFAULT": "DEFAULT",
+        "1": "Don't allow users to install and run apps from the Marketplace",
```
Comment (edyesed) [inline rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py:15]: thank you google

File: rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        "2": "Allow users to install and run any app from the Marketplace",
+        "3": "Allow users to install and run only selected apps from the Marketplace",
```
Comment (edyesed) [inline rules/gsuite_activityevent_rules/google_workspace_apps_marketplace_allowlist.py:17]: I think you could drop the `" from the marketplace"` bit of this to shorten it a little, though I suspect these are the values provided by google.

**Review by edyesed — APPROVED** [review]
detection for google workspace installable apps setting changed

**Review by calkim-panther — APPROVED** [review]

**Pull request merged**